docs/sources/next/javascript-api/k6-experimental/browser/request/_index.md

```diff
@@ -13,22 +13,22 @@ The `request` that the browser performs can be retrieved from the [Response](htt
+| [request.frame()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/frame)                                       | The [Frame](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/frame/) that initiated the request   |
+| [request.headers()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/headers)                                   | Returns an object of headers associated to the request.                                                                          |
+| [request.headersArray()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/headersarray)                         | An array with all the request HTTP headers                                                                                       |
+| [request.headerValue(name)](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/headervalue)                       | Returns the value of the header matching the name. The name is case insensitive.                                                 |
+| [request.isNavigationRequest()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/isnavigationrequest)           | Returns a boolean stating whether the request is for a navigation.                                                               |
+| [request.method()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/method)                                     | Request's method (GET, POST, etc.).                                                                                              |
+| [request.postData()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/postdata)                                 | Contains the request's post body, if any.                                                                                        |
+| [request.postDataBuffer()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/postdatabuffer)                     | Request's post body in a binary form, if any.                                                                                    |
+| [request.resourceType()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/resourcetype)                         | Contains the request's resource type as it was perceived by the rendering engine.                                                |
+| [request.response()](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/request/response)                                 | Returns the matching [Response](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/response) object |
```

Comment:
Missing "."

Reply:
👍 Updated in af3bc5a5ef3c71ef22fe52bcd7b6ec4f3cf37fbd